platform/src/test/kotlin/org/stellar/anchor/platform/custody/fireblocks/FireblocksPaymentServiceTest.kt

```diff
@@ -155,4 +156,25 @@ class FireblocksPaymentServiceTest {
 
     JSONAssert.assertEquals(responseJson, gson.toJson(response), JSONCompareMode.STRICT)
   }
+
+  @Test
+  fun test_getTransactionsByTimeRange_success() {
+    val responseJson =
+      getResourceFileAsString("custody/api/payment/fireblocks/get_transactions_response.json")
+    val urlCapture = slot<String>()
+    val queryParamsCapture = slot<Map<String, String>>()
+
+    every { fireblocksClient.get(capture(urlCapture), capture(queryParamsCapture)) } returns
+      responseJson
+
+    val startTime = Instant.now().minusSeconds(5)
+    val endTime = Instant.now()
+    val response = fireblocksPaymentService.getTransactionsByTimeRange(startTime, endTime)
+
+    Assertions.assertEquals("/v1/transactions", urlCapture.captured)
+    Assertions.assertEquals(3, queryParamsCapture.captured.size)
+
+    Assertions.assertEquals(2, response.size)
```

Comment:
Looks like this assert is redundant, as the json size will be validated in the next asset